### ISSUE TITLE
files not visible in mac docker

### ISSUE THREAD

**Issue by kateharline** [hci-unihd/plant-seg#103]
Hi, I'm using the mac docker image on Monterey 12.0.1, I'm able to get the PlantSeg GUI to load with
% docker run -it --rm -v PATH_TO_DATASET:/Users/kateharline/workspace/20210627   -e DISPLAY=10.48.88.110:0 wolny/plantseg

However, when I try to navigate to the directory with my image files or load an individual file, none are visible. The log reads
2021-11-30 18:17:02,546 - Executing pipeline step: 'preprocessing'. Parameters: '{'state': True, 'save_directory': 'PreProcessing', 'factor': [1.0, 1.0, 1.0], 'order': 2, 'crop_volume': '[:, :, :]', 'filter': {'state': False, 'type': 'gaussian', 'filter_param': 1.0}}'. Files [].

and it throws
AssertionError: Input file paths cannot be empty 

(Fair enough) 

I have tried this with my image files as .tif and .h5

**Comment by bipashadey** [comment]
I have the same error! any leads?

**Comment by wolny** [comment]
Hi @kateharline, thanks for raising the issue. 
I see the error in your docker command: you've incorrectly mounted your files inside the docker container. As explained in the documentation (https://github.com/hci-unihd/plant-seg#mac-os) the `PATH_TO_DATASET`  is the directory from host which you want to mount. So it should be `/Users/kateharline/workspace/20210627`. The second argument (after the colon) is the directory inside the container, e.g. `/root/datasets`. This is where you should navigate the PlantSeg GUI to load the files.

So basically your command should be:

`docker run -it --rm -v /Users/kateharline/workspace/20210627:/root/datasets -e DISPLAY=10.48.88.110:0 wolny/plantseg`

and then you should navigate to the `/root/datasets` to find your files.
